libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -277,8 +277,13 @@ void AP_ADSB::detect_instance(uint8_t instance)
 
     case Type::Sagetech:
 #if HAL_ADSB_SAGETECH_ENABLED
-        if (AP_ADSB_Sagetech::detect()) {
-            _backend[instance] = new AP_ADSB_Sagetech(*this, instance);
+        // gcs().send_text(MAV_SEVERITY_WARNING, "ADSB: Checking for Sagetech.");
+        // if (AP_ADSB_Sagetech::detect()) {
+        //     _backend[instance] = new AP_ADSB_Sagetech(*this, instance);
+        //     return;
+        // }
```

Comment:
So the reason for this was that ideally we should be able to define a new ADSB_TYPE parameter "Sagetech MXS", but I wasn't able to get that to work, so I overrode the existing Sagetech ADSB_TYPE. I didn't know if this is what we want to do long term or if this would be just temporary which is why this is left commented in case I need to revert it. Let me know what you think on this.

Reply:
Okay, we can leave them in for know while we investigate adding a new type.